hms-api/serverless.local.yml

```diff
@@ -546,5 +585,31 @@ resources:
                 KeyType: HASH
               - AttributeName: userId
                 KeyType: RANGE
+            Projection:
+              ProjectionType: ALL
+
+    commentTable:
+      Type: AWS::DynamoDB::Table
+      DeletionPolicy: Retain
+      Properties:
+        TableName: ${self:provider.environment.IDEA_COMMENT_TABLE}
+        BillingMode: PAY_PER_REQUEST
+        AttributeDefinitions:
+          - AttributeName: id
+            AttributeType: S
+          - AttributeName: ideaId
+            AttributeType: S
+          - AttributeName: ownerId
+            AttributeType: S
+        KeySchema:
+          - AttributeName: id
+            KeyType: HASH
+        GlobalSecondaryIndexes:
+          - IndexName: ${self:provider.environment.IDEA_COMMENT_BY_IDEA_ID_INDEX}
+            KeySchema:
+              - AttributeName: ideaId
+                KeyType: HASH
+              - AttributeName: id
+                KeyType: RANGE
             Projection:
```

Comment:
it sounds weird, but end the file with a newline please... it's a weird thing about how strings end in C... and it will shut github up about it